src/style/mixins.scss

```diff
@@ -261,8 +264,22 @@
       fill: currentColor;
     }
 
+    :disabled &,
+    [aria-disabled='true'] & {
+      background-color: var(--iui-color-background-2);
+      fill: var(--iui-icons-color-actionable-disabled);
+
+      @media (forced-colors: active) {
+        fill: GrayText;
+      }
+    }
+
     .iui-disabled img#{&},
-    .iui-disabled div#{&} {
+    .iui-disabled div#{&},
```

Comment:
so these classes also not needed?

Reply:
I'm anticipating this mixin being used in other places (menu for example) which are still using `.iui-disabled`.  But since header is the only one using this mixin at the moment, I'll remove it.